src/components/Main.js

```diff
@@ -1,64 +1,37 @@
 import {Component} from 'react'
+import BeastImage from './HornedBeast'
 import Container from 'react-bootstrap/Container'
 import Row from 'react-bootstrap/Row'
-import Col from 'react-bootstrap/Col'
-import Card from 'react-bootstrap/Card'
+
 
 
 
 class Main extends Component {
 
+
+
   render() {
```

Comment:
This all looks really great! Just want to try and clean up a lot of extra space from files, keeping things nice and clean 👍